### PR TITLE
unify schema naming

### PR DESCRIPTION
**Pull request by saidkarim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Important notice: In order to guarantee compatibility between S64 DA and s64da-b
 
    | Parameter      | Description                                            |
    | -------------- | ------------------------------------------------------ |
-   | `schema`       | The schema to deploy. Schemas are directories the benchmarks/\<benchmark\>/schemas directory and start with either `sdb_`, `s64da_`, or `psql_`. The schema name equals the directory name. |
+   | `schema`       | The schema to deploy. Schemas are directories in the benchmarks/\<benchmark\>/schemas directory and each benchmark has the following schemas: <br> - `psql_native`: plain tables + keys + indexes;<br> - `s64da_native`: as psql_native but with CREATE EXTENSION swarm64da (only default enabled features);<br> - `s64da_native_enhanced`: as s6da_native + opt-in features: columnstore index; <br> - `s64da_performance`: whatever needed to give the best S64 DA runtime. <br>The schema name equals the directory name. |
```
Comment (lmcarril) [inline README.md:41]: As this is going to be public the phrasing should be better:
`psql_native`: standard PostgreSQL schema;
`s64da_native`: as `psql_native` but with S64 DA enabled and only its default features;
`s64da_native_enhanced`: as `s6da_native`but with using some S64 DA opt-in features, such as the columnstore index;
`s64da_performance`: schema that provides the best performance for S64 DA.

**Comment by lmcarril** [comment]
In SSB s64da_native_enhanced: foreign keys, primary keys and indexes are missing. I would also remove "CREATE STATISTICS", because it is something not so obvious and it is not in the other native schemas of SSB.

**Comment by lmcarril** [comment]
In TPCH s64da_native and s64da_native_enhanced all primary keys, foreign keys and postgres indexes are missing. From s64da_native_enhanced I would also remove "CREATE STATISTICS"

And similarly in TPCDS

**Comment by lmcarril** [comment]
Another suggestions:
- maybe S64 DA specific stuff should be in specific sql files when possible, if postgres indices have their own file, the columnstore indices could also be in some: s64da.sql or s64da_indexes.sql
- reusing sql files between schemas *could* reduce maintenance
- maybe the indexes.sql file should be renamed to psql_indexes.sql to clarify that no s64da index is present there

File: README.md (hunk 1 above)
Comment (lmcarril) [reply README.md:41]: See also other comments in the PR

**Comment by lmcarril** [comment]
Suggestions:
    reusing sql files between schemas could reduce maintenance, instead of needing to have copies of them
    maybe the indexes.sql file should be renamed to psql_indexes.sql to clarify that no s64da index is present there

File: s64da_benchmark_toolkit/prepare.py [hunk 2]
```diff
@@ -261,7 +261,7 @@ def get_ingest_tasks(self, table):
         return []
 
     def add_indexes(self):
-        for sql_file in ('primary-keys.sql', 'foreign-keys.sql', 'indexes.sql'):
+        for sql_file in ('primary-keys.sql', 'foreign-keys.sql', 'indexes.sql', 's64da_indexes.sql'):
```
Comment (lmcarril) [inline s64da_benchmark_toolkit/prepare.py:264]: @saidkarim here I added the new file with just the columnstore indexes

**Comment by lmcarril** [comment]
After finishing this one, we should give another pass to the Appsflyer schema in the internal repo, so it also contains a pre_schema.sql and a s64da_indexes.sql

**Review by oferz — APPROVED** [review]

**Pull request merged**